gatsby-node.js

```diff
@@ -51,4 +51,11 @@ exports.onCreateNode = ({ actions, node, getNode }) => {
 
 exports.onCreatePage = ({ page, actions }) => {
   createIntlPages({ page, actions })
+
+  // create page /blog-cn/* for redirection
+  const { createPage } = actions
+  if (page.path.match(/^\/blog-cn/)) {
```

Comment:
I think it's better to bump this section into a module. Like the top imports of `gatsby-node.js`.

Reply:
Yes, it would be clear to import this as a module.